src/GLdispatch/vnd-glapi/entry_aarch64_tsd.c

```diff
@@ -68,6 +68,35 @@
  * The 'found_dispatch' section computes the correct offset in the dispatch
  * table then does a branch without link to the function address.
  */
+#if defined(__CHERI_PURE_CAPABILITY__)
+#define STUB_ASM_CODE(slot)                           \
+    "stp c1, c0, [csp, #-32]!\n\t"                    \
+    "adrp c0, :got:_glapi_Current\n\t"                \
+    "ldr c0, [c0, #:got_lo12:_glapi_Current]\n\t"     \
+    "ldr c0, [c0]\n\t"                                \
+    "gcvalue x0, c0\n\t"                              \
```

Comment:
That's not what I meant, clearly that doesn't work because 11f expects c0 to still be a valid capability. What I mean is that instead of doing `gcvalue xM, cN` you can delete the instruction and substitute xM with xN (provided cN remains the same capability, which it obviously is here).